.github/workflows/update-catalog.yaml

```diff
@@ -0,0 +1,39 @@
+name: Generate FBC catalogs
+
+on:
+  pull_request:
+    types: [opened, edited, reopened, synchronize]
```

Comment:
Does this run in forks? If so, can we restrict it to only run in the main repo?

Reply:
How is it configured?